### ISSUE TITLE
Background image scrolling problem.

### ISSUE THREAD

**Issue by Seejay01** [CDrummond/cantata#1554]
If the queue style is set to 'Table' and 'Current album cover' is selected as the background image, the background image glitches out while scrolling the queue.

1. Select Style: 'Table' in the Play Queue tab under Interface Settings
2. Select 'Current album cover' as the background image.
3. Fill the queue with enough tracks so that scrolling is required.
4. Scroll
5. See error

**Expected behavior**
as with 'Grouped Albums' selected, the background should remain static while scrolling the queue.

**Version Info**
 - OS: Linux
 - Cantata Version: 2.4.0

I'm using Cinnamon DE on Arch Linux on my PC and LXDE on my laptop. Both have the same issue. I don't recall having this issue when I was using KDE Plasma. Might have something to do with GTK vs QT?

**Comment by CDrummond** [comment]
Also seems to affect the grouped queue. :(

**Comment by CDrummond** [comment]
Feature has been disabled, as I have no time to look into a real fix.

**Comment by CDrummond** [comment]
Jsut in case anyone wants to look into this, to re-enable this feature edit `config.h.cmake` and change `#define ENABLE_VIEW_BACKGROUND (QT_VERSION < QT_VERSION_CHECK(5, 12, 0))` to `#define ENABLE_VIEW_BACKGROUND 1`

**Comment by oyvinds** [comment]
I was immensely disappointed to find that the background album cover images were removed when I started Cantata 2.4.0. I tested re-enabling the feature and it was fine, no issues. My system has Qt 5.13.2. I made a brief video of the main Cantata 2.4.0 window on my system:
https://yoona.everdot.org/linux/Cantata/cantata-2.4.0.webm
I don't see any glitches.

Perhaps the #define line disabling this feature could be changed to be Qt-version specific?
